packages/ui/src/components/va-dropdown/VaDropdown.vue

```diff
@@ -296,4 +296,8 @@ export default class VaDropdown extends mixins(
     z-index: var(--va-dropdown-content-wrapper-z-index);
   }
 }
+
+.va-dropdown:not(.va-popover) {
```

Comment:
If you want to create and exception for popover (dropdown width shouldn't be 100%), then it's better to be done in context of Popover, as dropdown is Popover dependency.
Dropdown shoudn't even know about Popover existance.